src/map/status.cpp

```diff
@@ -3139,6 +3285,12 @@ static int status_get_hpbonus(struct block_list *bl, enum e_status_bonus type) {
 				bonus += sc->data[SC_UPHEAVAL_OPTION]->val2;
 			if(sc->data[SC_LAUDAAGNUS])
 				bonus += 2 + (sc->data[SC_LAUDAAGNUS]->val1 * 2);
+#ifdef RENEWAL
+			if (sc->data[SC_ANGELUS])
+				bonus += sc->data[SC_ANGELUS]->val1 * 50;
```

Comment:
Skill Angelus
It doesn't increase as percentage.  
MaxHP + 50 per level.

Reply:
Did you apply this manually? Make sure it's under the `STATUS_BONUS_RATE` clause.

Reply:
When it's under the STATUS_BONUS_RATE clause , we got +500% hp bonus.
But it should be +50 per skill level.
Sorry , my English is weak . I don't know how to description it

Reply:
You're right! Sorry. I looked at the DEF part of the skill description.